code/controllers/subsystems/initialization/materials.dm

```diff
@@ -146,6 +146,21 @@ SUBSYSTEM_DEF(materials)
 		global.default_strata_type_by_z[s_key] = pick(subtypesof(/decl/strata))
 	return global.default_strata_type_by_z[s_key]
 
+/datum/controller/subsystem/materials/proc/get_material_by_name(var/mat_name)
+	RETURN_TYPE(/decl/material)
```

Comment:
What does this do here? Clearly this proc can return a decl, null, or 0 just by inspecting the code, and yet we are not getting a compile-time error.

Reply:
Linter told me to, if I'm being honest.